include/boost/url/impl/authority_view.ipp

```diff
@@ -256,7 +256,7 @@ parse_authority(
     if(s.size() > authority_view::max_size())
         detail::throw_length_error(
             "authority_view::max_size exceeded",
-            BOOST_CURRENT_LOCATION);
+            BOOST_URL_POS);
```

Comment:
If Peter does it this way, why can't we? https://github.com/boostorg/system/blob/cc7c2f7ee42623aef41c8cb45fe8d13f7bc87d7f/include/boost/system/result.hpp#L236

He puts the macro in the parameter list as a default argument... can we do that in `throw_length_error`?